### DISCUSSION TITLE
How to set a fixed width ?

### DISCUSSION THREAD

**Discussion by AndrewsGomesViamo** [Akryum/v-tooltip#873] [Q&A]
if I add a long text, the image look like this. using v-tooltip directive.

![image](https://user-images.githubusercontent.com/98570842/176225589-030f4996-f936-472e-b5c3-8b906de6f2c6.png)

I've tried this to fix

```
Vue.use(FloatingVue, {
  // Hide on mouseover tooltip
  autoSize: true,
}
```
That's what I received

![image](https://user-images.githubusercontent.com/98570842/176225926-ae828088-b907-400e-b455-d28099db2b2e.png)

How to set a fixed width ?

**Comment by JokerMartini** [comment]
yeah i wonder the same how to set a fixed width or even something like 50vw
